apps/files_sharing/lib/DefaultPublicShareTemplateProvider.php

```diff
@@ -226,11 +227,24 @@ protected function addOpenGraphHeaders(IShare $share): void {
 			$ogPreview = $this->urlGenerator->getAbsoluteURL($this->urlGenerator->imagePath('core', 'favicon-fb.png'));
 		}
 
+		$description = $this->defaults->getName() . ($this->defaults->getSlogan() !== '' ? ' - ' . $this->defaults->getSlogan() : '');
+
```

Comment:
```suggestion
$title = $shareNode->getName();
```
And reuse `$title` for bot OG and Twitter title headers.

Reply:
I chose not to reuse it, as it's just a simple method call, whereas the description has some logic associated with it that I didn't want to duplicate.

but I can pull it out into a variable if you'd like.